addons/account/models/mail_tracking_value.py

```diff
@@ -1,15 +0,0 @@
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import api, models
-
-
-class MailTrackingValue(models.Model):
-    _inherit = 'mail.tracking.value'
-
-    @api.ondelete(at_uninstall=True)
-    def _except_audit_log(self):
-        self.mail_message_id._except_audit_log()
```

Comment:
This means that during migration, all the messages linked to tracking values **HAVE TO** be updated to contain the old tracking values in the body.